README.md

```diff
@@ -2,6 +2,15 @@
 
 This is Openembedded layer of [thin-edge.io](https://github.com/thin-edge/thin-edge.io).
 
+## What is and isn't supported?
+
+- which init managers
+- what package format (deb, rpm, apk)
+- versions of thin-edge
+- versions of Yocto
+- what rust versions are required
+- which versions of optional components
```

Comment:
todo: this was meant to be filled in later, my guess is that's not all that is probably out of date in this README

Reply:
README needs overhaul as most of the things are outdated.